modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/computation/BulkScrollerComputation.java

```diff
@@ -111,10 +114,18 @@ public BulkScrollerComputation(String name, int nbOutputStreams, int scrollBatch
     // @since 11.2
     public BulkScrollerComputation(String name, int nbOutputStreams, int scrollBatchSize, int scrollKeepAliveSeconds,
             Duration transactionTimeout, boolean produceImmediate) {
+        this(name, nbOutputStreams, scrollBatchSize, scrollKeepAliveSeconds, DEFAULT_SCROLL_TRANSACTION_TIMEOUT,
+                produceImmediate, 0);
+    }
+
+    // @since 11.4
+    public BulkScrollerComputation(String name, int nbOutputStreams, int scrollBatchSize, int scrollKeepAliveSeconds,
+            Duration transactionTimeout, boolean produceImmediate, int produceImmediateThreshold) {
```

Comment:
Isn't it worth considering a builder pattern, given the number of parameters that we now pass at construction?

Reply:
maybe, but this constructor is only used in one place and is not intended to be used elsewhere